### DISCUSSION TITLE
Reduce `pac solution` noise

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/powerplatform-vscode#752] [Ideas]
pac will often output multiple lines rather than updating the existing line, e.g. for execution time and percentage complete.  This happens mainly in the `solution` and `deploy` suite of commands.  It would be good to remove the noise here.  Some options:

1) Remove the blank lines between updates - so doesn't scroll off the top as quick
2) Remove percentage complete (because it doesn't ever reach 100%) and just show execution time
3) When running interactively, update the same line on the terminal, rather than outputting a new line.  In a similar behaviour to `npm install` or `winget`

Example:

![image](https://github.com/microsoft/powerplatform-vscode/assets/6078398/aa591067-03aa-4acd-9bff-c8ddc83de71c)

**Comment by petrochuk** [comment]
Moving to discussions to allow voting
